docs/contribute/doc.md

```diff
@@ -117,6 +122,46 @@ else 0
 The `#doc_blame` command can be run at the bottom of a file to list all definitions that do not have
 doc strings. `#doc_blame!` will also list theorems and lemmas.
 
+## Tactic doc entries
+
+Interactive tactics, user commands, hole commands and user attributes should have doc strings
+explaining how they can be used. The `add_tactic_doc` command should be invoked afterwards to add a
+doc entry to the appropriate page in the online docs.
+
+Example:
+```lean
+add_tactic_doc
+{ name := "display name of the tactic",
+  category := cat,
+  decl_names := [`dcl_1, dcl_2],
+  tags := ["tag_1", "tag_2"],
+  description := "describe what the command does here"
+}
+```
+
+The argument to `add_tactic_doc` is a structure of type `tactic_doc_entry`.
+* `name` refers to the display name of the tactic; it is used as the header of the doc entry.
+* `cat` refers to the category of doc entry.
+  Options: `doc_category.tactic`, `doc_category.cmd`, `doc_category.hole_cmd`, `doc_category.attr`
+* `decl_names` is a list of the declarations associated with this doc. For instance,
+  the entry for `linarith` would set ``decl_names := [`tactic.interactive.linarith]``.
+  Some entries may cover multiple declarations.
+  It is only necessary to list the interactive versions of tactics.
+* `tags` is an optional list of strings used to categorize entries.
+* `description` is the body of the entry. Like doc strings, it can be formatted with markdown.
+  What you are reading now is taken from the description of `add_tactic_doc`.
+
+If only one related declaration is listed in `decl_names` and it does not have a doc string,
+`description` will be automatically added as its doc string. If there are multiple declarations, you
+can select the one to be used by passing a name to the `inherit_description_from` field.
+
+If you prefer a tactic to have a doc string that is different then the doc entry, then between
+the `/--` `-/` markers, write the desired doc string first, then `---` surrounded by new lines,
```

Comment:
I don't feel too strongly either way. On one hand, it might be confusing to see "doubled" documentation. On the other hand, it might save users a click to the doc pages if we leave it alone.

Reply:
I think the doc pages should certainly remove the part before. I'm ambivalent whether VSCode should remove the part after. It will usually be hidden by the scroll bar by default, and as Bryan says, maybe the info could be useful?